pom.xml

```diff
@@ -27,20 +27,59 @@
     <folio-spring-base.version>1.0.0</folio-spring-base.version>
     <openapi-generator.version>4.3.1</openapi-generator.version>
     <mapstruct.version>1.3.1.Final</mapstruct.version>
-    <json.version>20200518</json.version>
+    <es-client.version>7.10.0</es-client.version>
+    <spring-kafka.version>2.6.3</spring-kafka.version>
+    <apache-commons-io.version>2.6</apache-commons-io.version>
+    <apache-commons-collections.version>4.4</apache-commons-collections.version>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>org.folio</groupId>
-      <artifactId>folio-spring-base</artifactId>
-      <version>${folio-spring-base.version}</version>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-configuration-processor</artifactId>
```

Comment:
Why aren't we using folio-spring-base?

Reply:
For now folio-spring-base cannot be used without database connection, which is not needed for now in search. I asked @TarasSpashchenko to fix it. When it is fixed, in will be brought back.